src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -47,32 +81,103 @@ While _written language_ and _character_ are slightly less precise, they should
 
 Place exceptions, edge cases, and other kinds of supplemental information right next to the primary content they're related to.
 
 ## Use inclusive images
-Always provide [alt text](/image-alt).
 
-If you're creating an illustration, the parts that are essential for understanding the illustration should have a contrast ratio of at least 3:1. You can verify the contrast using checkers like the ones from [WeAIM](https://webaim.org/resources/contrastchecker/) or the [Paciello Group](https://developer.paciellogroup.com/resources/contrastanalyser/).
+Images are visual mediums of communicating with our users. As such, it's
+critical we share images in an inclusive and accessible manner.
+
+### Use appropriate contrast
+
+If you create an illustration or diagram, the content which is essential for
+the user's understanding should have a contrast ratio of at least 3:1. You can
+verify the image contrast using checkers:
+
+*  [WebAIM](https://webaim.org/resources/contrastchecker/)
+*  [Paciello Group](https://developer.paciellogroup.com/resources/contrastanalyser/)
+
+### Text associated with image
+
+You may find it useful to include [alt text](#alt-text) or figure captions.
+Independent of these elements, an introductory sentence should precede most
+images. The sentence can end with a colon or a period; usually a colon if it
+immediately precedes the image, usually a period if there's more material
+(such as a note paragraph) between the introduction and the image.
+
+You don't need to introduce screenshots that immediately follow procedural
+text that describes a UI.
 
-Avoid images that may exclude certain audience members. Just like when you _write_ about people, remember to be inclusive when you _show_ people and the things they do and make. For example, avoid stock photos that show only male developers. As an alternative, check out [WOCINTECH Stock Photos](https://www.wocintechchat.com/blog/wocintechphotos).
+### Alt text
+
+Alt text can be defined as a concise description of the image that can replace the image when the image isn't visible. Visibility can be related to use of assistive technologies (such as screen readers), preferences (such as use of text-only browsers), or circumstance (such as people with a low-bandwidth internet connection).
+
+Alt text should consider the context of the image, not just its content. For
+more information, see alt attribute.
+
+When appropriate, provide [alt text](/image-alt).
+
+Consider the following when writing alt text:
+
+*  Don't include phrases like "Image of" or "Photo of" or "Screenshot of".
+*  Include punctuation. When screen readers encounter punctuation, they pause
+   before continuing.
+*  Write short, descriptive alt text in 155 characters or less. Some screen
+   readers cut off reading alt text when longer than 155 characters.
+
+Review more [alt text best
+practices](https://developers.google.com/style/images#alt-text).
+
+### Avoid words in images
```

Comment:
Another excellent addition! I often call this out when editing, it's nice to have a reference to share in feedback to back up my reasoning.